doc/source/release-notes/Percona-Server-5.5.61-38.13.rst

```diff
@@ -0,0 +1,37 @@
+.. rn:: 5.5.61-38.13
+
+================================================================================
+|Percona Server| 5.5.61-38.13
+================================================================================
+
+Percona is glad to announce the release of |Percona Server| 5.5.61-38.13 on
+August 17th, 2018. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.5/Percona-Server-5.5.61-38.13/>`_
+and from the :doc:`Percona Software Repositories </installation>`. This release
+merges changes of :ref:`MySQL 5.5.61
+<http://dev.mysql.com/doc/relnotes/mysql/5.5/en/news-5-5-61.html>", including
+all the bug fixes in it. Percona Server for MySQL 5.5.61-38.13 is now the
+current GA release in the 5.6 series. All of Percona’s software is open-source
+and free.
+
+Bugs Fixed
+================================================================================
+
+* The ``--innodb-optimize-keys`` option of the ``mysqldump`` utility fails when
+  a column name is used as a prefix of a column which has the AUTO_INCREMENT
```

Comment:
when a table dumped has two columns, one of which is named with a prefix of another, and the other one has the AUTO_INCREMENT attribute